standalone/src/models/custom-application/custom-application-menu-link/custom-application-menu-link-draft/generator.ts

```diff
@@ -1,8 +1,8 @@
 import { fake, Generator } from '@/core';
 import { LocalizedField } from '@/models/commons';
-import type { TCustomApplicationMenuLinkDraft } from '../types';
+import type { TCustomApplicationMenuLinkDraftGraphql } from './types';
```

Comment:
We should be removing this file, shouldn't we?

Reply:
fixed in https://github.com/commercetools/test-data/pull/827/commits/0560e3787880fb668a77fcea122e3cac3b6e5826